src/wp-includes/update.php

```diff
@@ -2,19 +2,19 @@
 /**
  * A simple set of functions to check our version 1.0 update service.
  *
- * @package WordPress
+ * @package ClassicPress
  * @since WP-2.3.0
  */
 
 /**
- * Check WordPress version against the newest version.
+ * Check ClassicPress version against the newest version.
```

Comment:
Most of this file should still say `WordPress` until we actually switch over the update mechanism.